packaging/rpm/cvmfs-release.spec

```diff
@@ -70,10 +73,20 @@ fi
 if  [[ "$ID" == "fedora" ]]; then
 sed -i 's/EL/fedora/g' /etc/yum.repos.d/cernvm.repo
 fi
+if  [[ "$ID" == "almalinux" ]]; then
```

Comment:
What about rocky?  I think it's best to check for `" $ID_LIKE "` containing `"* rhel *"`.